File: Basics/First-Steps-In-Coding/Exercises/Resources/deposit_calculator.py
deposited_sum = float(input())
deposit_term = int(input())
interest_rate = float(input())
interest = deposited_sum * (interest_rate * 0.01)
month_interest = interest / 12
end_sum = deposited_sum + (deposit_term * month_interest)
print(end_sum)

File: JavaScript/Basics/0.1-First-Steps-In-Coding/Exercise/deposit-calculator.js
function depositCalculator(input) {
    let depositedSum = Number(input[0]);
    let depositTerm = Number(input[1]);
    let interestRate = Number(input[2]);
    let result = depositedSum + (depositTerm * ((depositedSum * interestRate / 100) / 12));
    console.log(result.toFixed(2));
}

depositCalculator(["200", "3", "5.7"]);
depositCalculator(["2350", "6", "7"]);
